jenkins/jenkins_home/jobs/pipeline1/config.xml

```diff
@@ -85,7 +89,8 @@ pipeline {
     }
     stage (&apos;Checkout the code and identify all Maven artifacts&apos;) {
       steps {
-        checkout scm: [$class: &apos;GitSCM&apos;, userRemoteConfigs: [[credentialsId: &apos;gitHubCredentials&apos;, url: repoUrl]], branches: [[name: branchName]]], poll: false
+        echo &quot;Creds=====&gt;${credentialsId} ${repoUrl} ${projectType}&quot;
```

Comment:
Let's keep the logging clean. It seems that by default, the `checkout` task will inform of the credential used:

```
...
The recommended git tool is: NONE
using credential simcgitlab
Cloning the remote Git repository
Cloning repository https://gitlab.tools.gbooking.ru/openmrs/ozone-simc.git
 > git init /var/jenkins_home/jobs/pipeline1/workspace # timeout=10
...
```

So we can probably just remove this logging line.